src/containers/shared/components/Transaction/EscrowCreate/Description.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { useTranslation, Trans } from 'react-i18next'
+import {
+  DATE_OPTIONS,
+  RIPPLE_EPOCH,
+  normalizeAmount,
+} from '../../../transactionUtils'
+import Account from '../../Account'
+import { localizeDate } from '../../../utils'
+import {
+  TransactionDescriptionComponent,
+  TransactionDescriptionProps,
+} from '../types'
+
+const Description: TransactionDescriptionComponent = (
+  props: TransactionDescriptionProps,
+) => {
+  const { t, i18n } = useTranslation()
+  const language = i18n.resolvedLanguage
+  const { data } = props
+  const cancelAfter = localizeDate(
+    (data.tx.CancelAfter + RIPPLE_EPOCH) * 1000,
+    language,
+    DATE_OPTIONS,
+  )
+  const finishAfter = localizeDate(
+    (data.tx.FinishAfter + RIPPLE_EPOCH) * 1000,
+    language,
+    DATE_OPTIONS,
+  )
+  return (
+    <>
+      {data.tx.Destination !== data.tx.Account ? (
+        <>
+          <Trans i18nKey="escrow_is_from">
+            The escrow is from
+            <Account account={data.tx.Account} />
+            to
+            <Account account={data.tx.Destination} />
+          </Trans>
+        </>
+      ) : (
+        <>
+          <Trans i18nKey="escrow_is_created_by">
```

Comment:
Ditto the translation difference - flagging for follow up based on what the plan is for that :)